extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -535,6 +538,13 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return {resolver, rejector, promise};
   }
 
+  /** @return {!Object<string,string|boolean|number>} */
+  getPageParameters_() {
+    return Object.assign(PAGE_LEVEL_PARAMS_, {
```

Comment:
can get rid of PAGE_LEVEL_PARAMS_ entirely, and move everything in here

Reply:
Done